app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.HomeFragment">
+
+    <Button
+        android:id="@+id/goToLearningButton"
+        android:layout_width="300dp"
+        android:layout_height="wrap_content"
+        android:onClick="launchToLearningActivity"
+        android:text="@string/main_btLearning"
+        app:layout_constraintBottom_toTopOf="@+id/goToDatasetsOverviewButton"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```

Comment:
MainActivity had a "Display current dataset" button. I think it was mainly for demo purposes though

Reply:
Yes I deleted it. I don't think there's any remnants of it right?

Reply:
Why did you delete it?

Reply:
Nicolas and I thought it made more sense to go through DatasetOverview and pick a dataset there. I can put it back if you want though.

Reply:
I wasn't aware of this design decision but now that I am I don't oppose it, so keep as is.